pkgs/development/python-modules/pwinput/default.nix

```
@@ -0,0 +1,29 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pytest,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "pwinput";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-yhqL0G4ohy11Hb1BMthjcSfCW0COo6NJN3MUpUkUJvM=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+  propagatedBuildInputs = [ pytest ];
+
+  doCheck = false;
+
+  meta = {
+    description = "A cross-platform Python module that displays **** for password input. Works on Windows, unlike getpass. Formerly called stdiomask.";
```

A lot of this information is irrelevant especially since Nix does not run on Windows
```suggestion
    description = "Python module that masks password input";
```